primitives/bridge/src/custom_do_process_message.rs

```diff
@@ -151,6 +151,7 @@ impl ConstantGasMeter {
     fn maximum_dispatch_gas_used_at_most(command: &Command) -> u64 {
         match command {
             Command::Test { .. } => 60_000,
+            Command::ReportRewards { .. } => 60_000,
```

Comment:
Minor: But add TODO here to revisit the gas cost value.

Reply:
_fixed_